queries.sql

```diff
@@ -7,4 +7,28 @@ SELECT date_of_birth FROM animals WHERE name='Agumon' OR name='Pikachu'
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5
 SELECT name FROM animals WHERE neutered = true
 SELECT name FROM animals WHERE name != 'Gabumon'
-SELECT name, weight_kg FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3
\ No newline at end of file
+SELECT name, weight_kg FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3
+
+/* DAY 2 */
+
+UPDATE animals SET species = 'unspecified' WHERE species is null;
+ROLLBACK
+UPDATE animals SET species = 'pokemon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'digimon' WHERE species is null;
```

Comment:
- [ ] Kindly correct this query. According to the requirements, all animals that end in `mon` should be updated by setting their species to `digimon` while the rest should be updated to `pokemon`. I think you have written this query the other way around. Kindly update this. 👍